core/core-impl/pom.xml

```diff
@@ -54,4 +54,47 @@
     </dependency>
 
   </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <artifactId>maven-failsafe-plugin</artifactId>
+        <configuration>
+          <skip>true</skip>
+        </configuration>
+
+        <executions>
+          <execution>
+            <id>token-validation</id>
+            <goals>
+              <goal>integration-test</goal>
+            </goals>
+            <configuration>
+              <skip>false</skip>
+              <excludes>
+                <exclude>NoKeycloakUrlSetSecuredIT</exclude>
+              </excludes>
+              <environmentVariables>
+                <LAUNCHER_KEYCLOAK_URL>${env.LAUNCHER_KEYCLOAK_URL}</LAUNCHER_KEYCLOAK_URL>
+              </environmentVariables>
+            </configuration>
+          </execution>
+          <execution>
+            <id>no-token-validation</id>
+            <goals>
+              <goal>integration-test</goal>
+            </goals>
+            <configuration>
+              <skip>false</skip>
+              <test>NoKeycloakUrlSetSecuredIT</test>
```

Comment:
@gastaldi this is how I run test where bypassing token validation is expected. WDYT?

Reply:
That looks fine. How about setting the environment var inside the test? Each test should already run in its own process afaik

Reply:
In theory, we can have a test which programmatically invokes this test class... but I am not sure that's a better solution, to be honest. What do we decide? Is that path good enough?

Reply:
If that works, I'm fine with this solution :+1: